peloton-to-garmin.py

```diff
@@ -158,6 +160,29 @@ def run(config):
 
             try:
                 title, filename, garmin_activity_type = tcx_builder.workoutSamplesToTCX(workout, workoutSummary, workoutSamples, config.output_directory)
+
+                device_info_file = Path('device_info.xml')
+                author_info_file = Path('author_info.xml')
+                if device_info_file.is_file() and author_info_file.is_file():
+                    logger.info("device_info and author_info found - fixing File: " + filename)
+
+                    with device_info_file.open("r") as file:
+                        deviceinfo = file.read()
+
+                    with author_info_file.open("r") as file:
+                        authorinfo = file.read()
+
+                    with open(config.output_directory+"/"+filename,"r") as file:
+                        filedata = file.read()
+
+                    filedata = filedata.replace(".0</","</")
+                    filedata = filedata.replace("</Activity>", deviceinfo + "</Activity>")
+                    filedata = filedata.replace("</TrainingCenterDatabase>",authorinfo + "</TrainingCenterDatabase>")
+                    filedata = filedata.replace("activityExtensions","ns3")
+                    filedata = re.sub(r"[.]\d+</Watts>","</Watts>", filedata)
+                    with open(config.output_directory + "/" +filename,"w") as file:
+                        file.write(filedata)
+
```

Comment:
In order to keep the code organized and easy to maintain, I propose a few changes here:

1. device_info_file & author_info_file should probably be defined in [lib/configuration.py](https://github.com/philosowaffle/peloton-to-garmin/blob/master/lib/configuration.py#L7)
2.  The logic to write author/device to the tcx file should occur in [lib/tcx_builder.py](https://github.com/philosowaffle/peloton-to-garmin/blob/master/lib/tcx_builder.py#L157) where all of the current TCX and xml writing occurs today.
3.  Instead of using find/replace, it would be cleaner to write the sections in a similar pattern to how its done in `tcx_builder.py`, taking advantage of the xml writer libraries.

I realize all of this is feedback on the original pr, not stuff you necessarily wrote, but it is ultimately why I had not merged that change yet because I was trying find time to come in and polish this stuff up first.

Reply:
I totally understand.  This is your project and your design.  You code it and put it where it makes sense in the overall design.  I simply just look for an opportunity to fix my own issue and then share what I found.  You do with it what you feel as you have to maintain it moving forward.

Reply:
Question for me: if you already want to shift to C# :) do we really want to spend time on this quick fix?

I've just done it for my last 100 peloton activities across types and verified successfuly that it counts to all challenges and badges and the import don't have any performance issues..

But that said - feel free to improve further.
I just wanted include the bare minimum on docs and sample files to make it useable for others.

Reply:
I mean, that's a fair point. Unfortunately, even with the shift to version 2, I will need to maintain support for version 1 for a bit and I want to make that as easy as possible for myself.  

Give me a couple days, if I don't have time to come in and refactor this by the end of the weekend then I will merge the pr as is, and refactor later.